### ISSUE TITLE
Loader not defined error when opening non .lds file extensions with ld-decode

### ISSUE THREAD

**Issue by chezsick** [happycube/ld-decode#151]
I Each times to execute ld-decode.py return
```
Traceback (most recent call last):
  File "./ld-decode/ld-decode.py", line 68, in <module>
    ldd = LDdecode(filename, outname, loader, frameoutput=foutput, system=system, doDOD = not args.nodod)
NameError: name 'loader' is not defined
```
How to fix this? or fix it
Thanks, Sick

**Comment by happycube** [comment]
The source file must have the correct file extension.  Most of the samples you'll see are .lds file type.  I'll fix that now to make it the default.

**Comment by simoninns** [comment]
Change issue title to reflect problem and labelled.

**Comment by chezsick** [comment]
but, how to read from hacktv?

**Comment by simoninns** [comment]
@chezsick - the input file must have a .lds extension at the moment in order to work with ld-decode.  Until this issue is resolved you should be able to simply rename your input file.  I'm not sure this will solve your issue (as I have no idea what hacktv file you're talking about, or it's contents).

**Comment by simoninns** [comment]
Closing this issue as it looks like any actual problem with ld-decode was solved in #151
